polaris/evaluate/_metric.py

```diff
@@ -65,17 +90,23 @@ class Metric(Enum):
     pearsonr = MetricInfo(fn=pearsonr, direction="max")
     spearmanr = MetricInfo(fn=spearman, direction="max")
     explained_var = MetricInfo(fn=explained_variance_score, direction="max")
+    absolute_average_fold_error = MetricInfo(fn=absolute_average_fold_error, direction="max")
```

Comment:
Based on our discussion in person, just adding this comment so we don't forget: We should check the `direction`!

Reply:
I believe the target is `1.0`, with a range of `[0, ∞)`.

Reply:
Actually, if targets are 0.0 there could be a division by zero. We should probably add a small delta to prevent that? Or check for it?